app/services/sessions.js

```diff
@@ -0,0 +1,12 @@
+const models = require('../models/index');
+const logger = require('../logger');
+
+module.exports.invalidateAll = async currentUser => {
+  try {
+    const currentSession = await models.session.findOne({ where: { userId: currentUser.id } });
+    return await currentSession.destroy();
```

Comment:
🔥 👀
Why did you take the time to add a table for sessions if you are going to delete them and not keep track of them? If you just wanted to delete them, adding a `session_id` in the user model was enough.
So now, since we have the model, lets work on not permanently destroying them. Sequelize can handle this for you, read about the `paranoid` option and `deletedAt` here: https://sequelize.org/master/manual/models-definition.html

Reply:
I added this table just to make it pretty and also i added this to handle multiple sessions (I didn't know what would come after ajaja)